webhook/env.go

```diff
@@ -37,3 +40,17 @@ func PortFromEnv(defaultPort int) int {
 	}
 	return port
 }
+
+func NameFromEnv() string {
+	if webhook := os.Getenv(webhookNameEnv); webhook != "" {
+		return webhook
+	}
+
+	panic(fmt.Sprintf(`The environment variable %q is not set
+If this is a process running on Kubernetes, then it should be using the downward
+API to initialize this variable via:
```

Comment:
This isn't the downward API 😂 

I think this is carryover from `system.Namespace()` 🤦

Reply:
Done some changes.PTAL